manifests/params.pp

```diff
@@ -94,4 +94,5 @@
   $environment_variables      = {}
   $config_variables           = {}
   $config_kernel_variables    = {}
+  $ulimit                     = 4096
```

Comment:
@cmurphy with this default you can't have more than (oversimplifying) 1K (connections + queues). Should you go above that limit, you either stop accepting new connections (as RabbitMQ does) or fail to write to disk (some tools are not prepared to handle failures like this) or simply shut down (e.g. Cassandra nodes typically shut down when they detect inability to do local disk I/O). It is often inadequate even for development and QA environments: many types of tests result in using a few hundreds of connections and a few thousands of queues.

By bumping it to 16K, you trade an insignificant amount of memory for a much lower chance of system unavailability or developer frustration.